Userland/Applications/Browser/Tab.cpp

```diff
@@ -135,6 +135,17 @@ Tab::Tab(BrowserWindow& window, Type type)
         view().set_focus(true);
     };
 
+    m_location_box->on_ctrl_return_pressed = [this] {
+        StringBuilder builder;
+        builder.append("www.");
+        builder.append(m_location_box->text());
+        builder.append(".com");
```

Comment:
Let's be a bit more flexible here: This strips leading and trailing periods, if present. Also, if there is still a period somewhere in the text, assume there is a TLD already present and don't append `.com`. This also makes this much more international, as this feature would otherwise be mostly limited to US users.
```cpp
builder.append("www.");
auto trimmed_text = m_location_box->text().trim("."sv, TrimMode::Both);
builder.append(trimmed_text);
if (!trimmed_text.contains("."sv)
    builder.append(".com");
```
Maybe you could also document what this does in the code, with a comment.

Reply:
Updated initial comment with specifics after talking to Max and used his code to develop the updated commit.